tests/skip_refinement.prm

```diff
@@ -0,0 +1,162 @@
+# A simulation with adaptive refinement only based on minimum refinement
+# function. As a consequence, refinement will not change the mesh
+# after a few steps. Confirm that ASPECT detects this and skips the
```

Comment:
When you talk about steps, I first thought of time steps (which of course does not make sense). 

```suggestion
# after this minimum refinement level is reached. Confirm that ASPECT detects this and skips the
```